setup.py

```diff
@@ -46,8 +46,10 @@
     "pyyaml>=3.13",
     "requests>=2.21.1",
     "scikit-learn>=0.20.2",
+    "scipy>=1.4.1",
```

Comment:
Are you able to work with scipy>=1.1.0 and statsmodels>=0.9.0?
These are the versions already in Azure Notebooks and would save us some install time if we didn't have to upgrade. AzureML should be ok - it has later versions of these.

Reply:
Zscore is available in older version of scipy but STL method was introduced statsmodels - 0.11.0 so will need upgrade. I will make changes for scipy and also add it to conda-reqs. 
- https://www.statsmodels.org/dev/release/version0.11.html
- https://github.com/statsmodels/statsmodels/pull/5926